lib/internal/test_runner/test.js

```diff
@@ -821,6 +821,10 @@ class Suite extends Test {
         return;
       }
 
+      if (this.parent?.hooks.before.length > 0) {
```

Comment:
Looking at line 814 it seems `this.parent` will always be defined: https://github.com/nodejs/node/pull/48877/files#diff-42583d1c2fb89ce93a6c2922840a13f8026995468ffe8b0ac8319d1749de6acbR814
Can we safely drop the optional chaining here as well?

Reply:
Thanks, done